src/openms/source/CHEMISTRY/ResidueDB.cpp

```diff
@@ -133,43 +132,42 @@ namespace OpenMS
     return s;
   }
 
-  void ResidueDB::setResidues_(const String& file_name)
+  void ResidueDB::initResidues_()
   {
     #pragma omp critical (ResidueDB)
     {
-      readResiduesFromFile_(file_name);
+      buildResidues_();
       buildResidueNames_();
     }     
   }
 
   void ResidueDB::addResidue_(Residue* r)
   {
     vector<String> names;
-    if (r->getName() != "")
+
+    // add name to lookup
+    if (r->getName() != "") 
```

Comment:
a residue without name? should that be allowed?

Reply:
I think these are modified residues. Not sure but I think they exist ;)